File: src/main/java/com/examples/testing/badextraprivatevariable/incorrect/java/extraprivatefield/Child.java
package com.examples.testing.badextraprivatevariable.incorrect.java.extraprivatefield;

import com.examples.testing.badextraprivatevariable.Parent;
import edu.illinois.cs.cs125.questioner.lib.Incorrect;

@Incorrect("features")
public class Child extends Parent {
  private final String firstName;
  private final String lastName;

  public Child(String setFirstName, String setLastName) {
    super(setLastName);
    lastName = setLastName;
    firstName = setFirstName;
  }

  @Override
  public String toString() {
    return firstName + " " + lastName;
  }
}

File: src/main/java/com/examples/testing/badextraprivatevariable/incorrect/kotlin/extraprivatefield/Child.kt
package com.examples.testing.badextraprivatevariable.incorrect.kotlin.extraprivatefield

import com.examples.testing.badextraprivatevariable.Parent
import edu.illinois.cs.cs125.questioner.lib.Incorrect

@Incorrect("features")
class Child(private val firstName: String?, private val lastName: String?) : Parent(lastName) {
  override fun toString() = "$firstName $lastName"
}